redeploy.sh

```diff
@@ -32,6 +32,16 @@ docker rmi hubmap/portal-ui:latest
 
 echo 'starting...'
 docker-compose -f \$COMPOSE_CONFIG up -d
+
+# Need to restart the hubmap-auth container on STAGE to avoid
+# portal.stage.hubmapconsortium.org and portal-prod.stage.hubmapconsortium.org get swapped
+if [ "$TARGET" = "stage" ]; then
+    echo 'Restart the hubmap-auth container... for STAGE only'
+    docker stop hubmap-auth
+    sleep 5
+    docker start hubmap-auth
```

Comment:
`sleep` always worries me: We're just asking for a race condition. Is there a reason not to use `docker restart`?

Reply:
I didn't even realize the existence of this `docker restart`. There's no reason not to use it. Feel free to use it.